tests/test_cache.py

```diff
@@ -0,0 +1,147 @@
+from django.template import Context
+
+from cms.api import add_plugin
+from cms.cache.placeholder import get_placeholder_cache
+from cms.test_utils.util.fuzzy_int import FuzzyInt
+
+from djangocms_alias.cms_plugins import Alias
+
+from .base import BaseAliasPluginTestCase
+
+
+class AliasCacheTestCase(BaseAliasPluginTestCase):
+    alias_template = """{% load djangocms_alias_tags %}{% render_alias plugin.alias %}"""
+
+    def setUp(self):
+        super().setUp()
+        # Create an alias and get a placeholder
+        self.alias = self._create_alias(published=True)
```

Comment:
This is redundant as they are specified in BaseAliasPluginTestCase. To ensure that no other changes were affecting the cache we had explicitly set these in the tests directly to ensure that no outer settings flawed our test.

Reply:
Removed setUp function